source/module_hamilt_general/module_surchem/test/support/rho_g

```diff
@@ -0,0 +1,12053 @@
+(2.58789e-07,4.56279e-08)
```

Comment:
This file is too large to be accepted as a supporting file in unit test.

Reply:
The file vloc is also too large to be accepted as a supporting file in unit test.